src/controller.rs

```diff
@@ -101,8 +111,19 @@ impl Controller for FoxBox {
         {
             Arc::get_mut(&mut self.upnp).unwrap().start().unwrap();
         }
+
+        if self.get_tls_enabled() {
+            let certificate_directory = PathBuf::from(
+                self.config.get_or_set_default("foxbox", "certificate_directory", "certs/"));
+
+            // If this fails, it just means that no certificates will be configured, which
+            // shouldn't cause a crash.
+            self.certificate_manager.reload_from_directory(certificate_directory).unwrap_or(());
```

Comment:
No, nothing is configured and the box won't respond to requests, this isn't too bad because once we've got LetsEncrypt support, we can update and add certs once it's already running.  Requiring stuff on disk to use TLS might be bad.